status-im/status-react
# Added assets icons and allow to filter tokens

fixes #2589
fixes #2606
fixes #2608

### Summary:

Add support for tokens and tokens filtering.

### Testing notes (optional):

Upgrades have to be tested.

### Steps to test:
- Open Status
- Browse wallet
- Some default token should be displayed
- Browse `manage assets`, go back
- Listed tokens should reflect your selection
- Restart app, listed tokens should be the same
- Step 3, etc.

status: ready
